packages/core/package.json

```diff
@@ -101,7 +101,9 @@
   },
   "devDependencies": {
     "@coinbase/wallet-sdk": "^3.4.1",
+    "@types/react": "^18.0.9",
     "@walletconnect/ethereum-provider": "^1.7.8",
+    "react": "^18.1.0",
```

Comment:
Just dependencies for testing serializing `React.createElement`.

Reply:
Don't love this, but it's a dev dependency. Could add a step to CI that [removes dev dependencies](https://github.com/pnpm/pnpm/issues/5695) so they don't show up on npm with `pnpm pkg delete devDependencies` (not super pressing though).